vmm/src/vm.rs

```diff
@@ -335,6 +335,15 @@ pub enum Error {
 
     #[error("Error locking disk images: Another instance likely holds a lock")]
     LockingError(#[source] DeviceManagerError),
+
+    #[error("Fw Cfg missing kernel file")]
```

Comment:
nit: `Fw Cfg missing kernel file` -> `Fw Cfg missing kernel`; you also don't have ` file` in the other message